pkg/client/client.go

```diff
@@ -751,7 +751,7 @@ func (c *Client) CreateOrUpdateIngress(ing *v1betaextensions.Ingress) error {
 
 func (c *Client) CreateOrUpdateAPIService(apiService *apiregistrationv1beta1.APIService) error {
 	apsc := c.aggclient.ApiregistrationV1beta1().APIServices()
-	_, err := apsc.Get(apiService.GetName(), metav1.GetOptions{})
+	apiService, err := apsc.Get(apiService.GetName(), metav1.GetOptions{})
```

Comment:
This looks like we won't be updating the object with our intended content, if we override the passed `apiService` with the one returned from the API. We will need some merging logic, I assume for the injected TLS cert from the serving certs controller.

Reply:
doh, you're totally right. This was only fixing the "obvious" case, namely just restarting the cluster monitoring operator pod, my bad :-/

Reply:
fixing this